rkt/api_service.go

```diff
@@ -193,10 +193,13 @@ func satisfiesPodFilter(pod v1alpha.Pod, manifest schema.PodManifest, filter v1a
 
 	// Filter according to the cgroup.
 	if len(filter.Cgroups) > 0 {
-		s := set.NewString(filter.Cgroups...)
-		if !s.Has(pod.Cgroup) {
-			return false
-		}
+                for _, cgroup := range filter.Cgroups {
+                        if strings.HasPrefix(cgroup, pod.Cgroup) {
```

Comment:
Shouldn't it be the reverse?
```
if strings.HasPrefix(pod.Cgroup, cgroup) {
```

api/v1alpha/api.proto would need to be updated:
```diff
 message PodFilter {
...
-        // If not empty, the pods whose cgroup are listed will be returned.
-        repeated string cgroups = 7;
+        // If not empty, the pods whose cgroup are prefixed as follows will be returned.
+        repeated string cgroupPrefix = 7;
```
But I don't know if we want to keep the API stable and avoid renaming fields.